.github/autobuild/linux_deb.sh

```diff
@@ -50,11 +53,18 @@ setup_cross_compiler() {
     sudo update-alternatives --install "/usr/bin/${ABI_NAME}-g++" g++ "/usr/bin/${ABI_NAME}-g++-${GCC_VERSION}" 10
     sudo update-alternatives --install "/usr/bin/${ABI_NAME}-gcc" gcc "/usr/bin/${ABI_NAME}-gcc-${GCC_VERSION}" 10
 
-    if [[ "${TARGET_ARCH}" == armhf ]]; then
-        # Ubuntu's Qt version only ships a profile for gnueabi, but not for gnueabihf. Therefore, build a custom one:
-        sudo cp -R "/usr/lib/${ABI_NAME}/qt5/mkspecs/linux-arm-gnueabi-g++/" "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/"
-        sudo sed -re 's/-gnueabi/-gnueabihf/' -i "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/qmake.conf"
-    fi
+    case "${TARGET_ARCH}" in
+        armhf)
+            # Ubuntu's Qt version only ships a profile for gnueabi, but not for gnueabihf. Therefore, build a custom one:
+            sudo cp -R "/usr/lib/${ABI_NAME}/qt5/mkspecs/linux-arm-gnueabi-g++/" "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/"
+            sudo sed -re 's/-gnueabi/-gnueabihf/' -i "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/qmake.conf"
+            ;;
+        arm64)
+            # Ubuntu's Qt version does not ship a profile for aarch64. Therefore, build a custom one based on gnueabihf:
+            sudo cp -R "/usr/lib/${ABI_NAME}/qt5/mkspecs/linux-arm-gnueabi-g++/" "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/"
+            sudo sed -re "s/arm-linux-gnueabi/${ABI_NAME}/" -i "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/qmake.conf"
+            ;;
```

Comment:
Hm.  I feel this isn't nice.  If the code is common for armhf and arm64 apart from the search/replace string, just set that based on TARGET_ARCH
```
local replace=""
[[ "${TARGET_ARCH}" == armhf ]] && replace='s/-gnueabi/-gnueabihf/'
[[ "${TARGET_ARCH}" == arm64 ]] && replace="s/arm-linux-gnueabi/${ABI_NAME}/"
[[ $replace != "" ]] && {
  # Ubuntu's Qt version only ships a profile for gnueabi, but not for gnueabihf. Therefore, build a custom one:
  sudo cp -R "/usr/lib/${ABI_NAME}/qt5/mkspecs/linux-arm-gnueabi-g++/" "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/"
  sudo sed -re $replace -i "/usr/lib/${ABI_NAME}/qt5/mkspecs/${ABI_NAME}-g++/qmake.conf"
}
```

Reply:
Good idea!
I've done that with the minor difference that I've kept the `case` for the `replace=` logic, mainly, because I'd expect the "shorthand if" to fail unless we added `|| true` which might hurt readability here?